planner/core/point_get_plan.go

```diff
@@ -95,6 +95,9 @@ func (p *PointGetPlan) ExplainInfo() string {
 			fmt.Fprintf(buffer, ", handle:%d", p.Handle)
 		}
 	}
+	if p.Lock {
+		fmt.Fprintf(buffer, ", lock")
```

Comment:
could you print more information about this `lock`?

Reply:
For example?